torch/_dynamo/variables/sdpa.py

```diff
@@ -65,7 +65,7 @@ def var_getattr(self, tx, name: str) -> VariableTracker:
             getattr_static(torch._C._SDPAParams, name)
         except AttributeError:
             # Using raise from is too verbose here
-            raise Unsupported(  # noqa: TRY200
+            raise Unsupported(  # noqa: B904
```

Comment:
Instead noqa here, do a `raise ... from None` to show that it's explicit according to the comment.